dts/bindings/pwm/nxp,s32-emios-pwm.yaml

```diff
@@ -0,0 +1,128 @@
+description: |
+  NXP S32 eMIOS PWM node for S32K3 SoCs. Each channel in eMIOS can be configured
```

Comment:
does this binding only applies for S32K3? I'd expect it can be reused in other SoCs like S32Z

Reply:
Right now is fit for S32K3 but yes can be extended, updated